tracer/test/Datadog.Trace.Security.Unit.Tests/EncoderUnitTests.cs

```diff
@@ -179,6 +179,15 @@ public void TestMapListDepth(int length, int expectedLength)
         Assert.Equal(expectedLength, CountNestedMapDepth(result));
     }
 
+    [SkippableTheory]
+    [InlineData(new object[] { "a", 3, new object[] { "b", 6 }, 44 }, "[ a, 3, [ b, 6 ], 44 ]")]
+    [InlineData(new object[] { "aba", "3", new object[] { new object[] { "w" }, "e" },  }, "[ aba, 3, [ [ w ], e ] ]")]
```

Comment:
I suggest you add some examples that include `null`, because I think the behaviour has changed in the refactoring...

e.g. is this _actually_ what you expect?

```suggestion
    [InlineData(new object[] { "aba", null, "3", new object[] { new object[] { "w" }, "e" },  }, "[ aba ]")]
```